build-library/action.yml

```diff
@@ -1,34 +1,49 @@
-name: "Build a Python library."
-description: "Build a Python library by generating wheels and sdist artifacts."
+name: >
+  Build library action
+
+description: >
+  Verifies if a Python library builds properly. As a result of a successfull
```

Comment:
```suggestion
  Verifies if a Python library builds properly. As a result of a successful
```

Reply:
I need to activate the spell checker of my text editor when working with documentation. Usually, I deactivate it because it complains about code syntax. However, activating it for doc-related tasks would avoid this typos.